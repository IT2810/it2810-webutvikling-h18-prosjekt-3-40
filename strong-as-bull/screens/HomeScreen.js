import React from 'react';

import {
    View,
    Text,
    StyleSheet,
} from 'react-native';
import {Thumbnail} from "native-base";

export default class HomeScreen extends React.Component{

    render(){

        return(
            <View style={styles.container}>
                <View style={styles.picture}>
                    <Thumbnail source={require('../assets/images/icon.png')} large style={styles.image} />
                </View>
                <Text style={styles.welcome}>
                    Welcome to
                </Text>
                <Text style={styles.name}>
                    Strong as Bull!
                </Text>

            </View>
        )
    }
}

const styles = StyleSheet.create({
    container: {
        backgroundColor: '#fff',
        flex: 1,
        paddingTop: 50,
    },
    name: {
        padding: 10,
        fontSize: 35,
        justifyContent: 'center',
        textAlign: 'center',
    },
    welcome: {
        textAlign: 'center',
        fontSize: 20,
        padding: 10,
    },
    picture: {
        alignItems: 'center',
    },
    image: {
        height: 200,
        width: 200,
    }
});