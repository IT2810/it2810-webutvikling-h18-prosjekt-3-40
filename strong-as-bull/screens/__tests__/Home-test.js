import 'react-native';
import React from 'react';
import HomeScreen from '../../screens/HomeScreen.js';
import renderer from 'react-test-renderer';

it('CreateContact snapshot', () => {
    const snap = renderer.create(
        <HomeScreen/>
    ).toJSON();

    expect(snap).toMatchSnapshot();

});
