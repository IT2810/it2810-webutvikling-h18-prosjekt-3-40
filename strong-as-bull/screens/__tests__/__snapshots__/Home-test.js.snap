// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateContact snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "paddingTop": 50,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <Image
      large={true}
      source={1}
      style={
        Array [
          Object {
            "borderRadius": 40,
            "height": 80,
            "width": 80,
          },
          Object {
            "height": 200,
            "width": 200,
          },
        ]
      }
      virtual={undefined}
    />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "padding": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 35,
        "justifyContent": "center",
        "padding": 10,
        "textAlign": "center",
      }
    }
  >
    Strong as Bull!
  </Text>
</View>
`;
